packages/core/src/types.ts

```diff
@@ -187,3 +187,21 @@ export type StoreContents = {
   context: Record<string, unknown>,
   breadcrumbs: BreadcrumbRecord[]
 }
+
+export type UserFeedbackForOptions = {
```

Comment:
Should this be `UserFeedbackFormOptions`?

Reply:
Yes, thanks!